core/src/main/java/org/infinispan/persistence/manager/PersistenceManagerImpl.java

```diff
@@ -160,6 +164,19 @@
       return null;
    }
 
+   @GuardedBy("lock#readLock")
+   private StoreStatus getStoreStatusLocked(Predicate<? super StoreConfiguration> predicate) {
+      StoreStatus firstStoreStatus = null;
+      for (StoreStatus storeStatus : stores) {
+         if (storeStatus.characteristics.contains(Characteristic.BULK_READ) &&
+             predicate.test(storeStatus.config)) {
+            firstStoreStatus = storeStatus;
```

Comment:
remove `firstStoreStatus` var and 
```suggestion
            return storeStatus;
```